### DISCUSSION TITLE
What's the best approach to adding css/js to landing pages

### DISCUSSION THREAD

**Discussion by acromatica** [breezefront/community#343] [1. Help] [answered]
What is the recommended way to add custom css and js to landing pages in Breeze? 
If the website has multiple landing pages, each with its own unique style (except for the standard header and footer shared across all the pages), should the styles and js be included in the bundle that's loaded on all pages, or should every page get its separate layout update?

`<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
        <css src="css/pages/custom-landing.css" />
    </head>
</page>
`

**Comment by vovayatsyuk** [comment, marked as answer]
It depends on the size of your style. If you are asking that, I assume it's not a few kilobytes :)

Here is a guide on how to create a custom CSS bundle: https://breezefront.com/docs/custom-styles#custom-css-bundle

Additionally, if it's a CMS page and you building it using Magento's PageBuilder, you can inline the CSS directly into this page using HTML element.
